torch/onnx/utils.py

```diff
@@ -481,14 +493,18 @@ def _export(model, args, f, export_params=True, verbose=False, training=False,
             proto, export_map = graph._export_onnx(
                 params_dict, opset_version, dynamic_axes, defer_weight_export,
                 operator_export_type, strip_doc_string, val_keep_init_as_ip, custom_opsets,
-                val_add_node_names)
+                val_add_node_names, val_use_large_model_format, model_file_location)
         else:
             proto, export_map = graph._export_onnx(
                 {}, opset_version, dynamic_axes, False, operator_export_type,
-                strip_doc_string, val_keep_init_as_ip, custom_opsets, val_add_node_names)
-
-        if enable_onnx_checker and operator_export_type != OperatorExportTypes.ONNX_ATEN_FALLBACK:
-            # Only run checker if enabled and we are not using ATEN fallback
+                strip_doc_string, val_keep_init_as_ip, custom_opsets, val_add_node_names,
+                val_use_large_model_format, model_file_location)
+
+        if enable_onnx_checker and \
+           operator_export_type is OperatorExportTypes.ONNX_ATEN_FALLBACK and \
+           not val_use_large_model_format:
```

Comment:
is onnx checker not ready for large model format?

Reply:
No, apparently ONNX checker does not support large models. I plan to open an ONNX issue for this.